backend/awsvpc/awsvpc.go

```diff
@@ -53,13 +53,42 @@ func (be *AwsVpcBackend) Run(ctx context.Context) {
 	<-ctx.Done()
 }
 
+type backendConfig struct {
+	RouteTableID interface{} `json:"RouteTableID"`
+}
+
+func (conf *backendConfig) routeTables() ([]string, error) {
+	if table, ok := conf.RouteTableID.(string); ok {
```

Comment:
wondering if we can just simplify the interface so that `RouteTableID` is always a slice, even if it has just one route table ID, that way we won't have to do this interface stuff. WDYT?